bigtable/client/table.h

```diff
@@ -224,6 +230,24 @@ class Table {
                          std::vector<Mutation> true_mutations,
                          std::vector<Mutation> false_mutations);
 
+  /**
+   * Return a sample of the row keys in the table, including approximate data
+   * sizes.
+   *
+   * @return a collection of SampleRowKeys. (The type of collection is specified
+   * by the user(vector or list).
+   */
+  template <template <typename...> class Collection>
+  Collection<Table::RowKeySample> sample_rows() {
+    Collection<Table::RowKeySample> result;
+    sample_rows_impl(
```

Comment:
Same here, should be `SampleRowsImpl()`.

Reply:
Done.